ops/static/css/fonts.css

```diff
@@ -4,47 +4,47 @@
   font-family: 'Raleway';
   font-style: normal;
   font-weight: 300;
-  src: local('Raleway Light'), local('Raleway-Light'), url(http://fonts.gstatic.com/s/raleway/v11/ZKwULyCG95tk6mOqHQfRBCEAvth_LlrfE80CYdSH47w.woff2) format('woff2');
+  src: local('Raleway Light'), local('Raleway-Light'), url(//fonts.gstatic.com/s/raleway/v11/ZKwULyCG95tk6mOqHQfRBCEAvth_LlrfE80CYdSH47w.woff2) format('woff2');
```

Comment:
Scheme-relative URLs are an anti-pattern, since anything that *can* be loaded via HTTPS *should* be loaded via HTTPS. That being the case, I suggest changing all these links to explicit HTTPS.